tests/test.c

```diff
@@ -113,6 +113,7 @@ struct Test tests[] = {
     TEST_CASE_EXPECTED(moreintegertests, 32),
     TEST_CASE_EXPECTED(send_receive, 18),
     TEST_CASE_EXPECTED(send_to_dead_process, 20),
+    TEST_CASE(selval),
```

Comment:
I'm amazed this was missing.

Reply:
I'm amazed this went unnoticed for so long: `!jump_to_address && (src_value == cmp_value)` :smile: